arm-cognitiveservices/2016-02-01-preview/swagger/cognitiveservices.json

```diff
@@ -238,7 +254,7 @@
           "CognitiveServicesAccounts"
         ],
         "description": "Returns all the resources of a particular type belonging to a resource group",
-        "operationId": "CognitiveServicesAccounts_ListByResourceGroup",
+        "operationId": "Accounts_ListByResourceGroup",
```

Comment:
Operation id -> ResourceTypePlural_OperationName will generate the API OperationName in a class ResourceTypePlural. So existing user side code will be break as now existing operation is changed. OperationId. I get that there is a error in PR build, consider these changes for the next API/SDK version.
In case your SDK is never released you can make these changes, as there is no user effected in that case.